script/book.rb

```diff
@@ -199,6 +203,19 @@ def save
       end
     end
   end
+
+  def book_v1_aliases(cs_number)
```

Comment:
So this follows the same style as the rest of this script.  And my reading of the script: this script generates all the static pages for the book. And deals with URLs.

In particular for redirects: we can see before this change that there is logic surrounding `front_matter` about aliases, and relevant to this PR, aliases to redirect from V1 links to the current book (V2).

Now this PR adds this function which is effectively a thin wrapper around the YAML files of the out-in-the-wild (Wayback Machine) URLs.

Makes sense.